--- a/webview/assets/app-initial~app-main~worktree-init-v2-page~remote-conversation-page~pull-requests-page~new-~mam8fk3d-QYu6Tutl.js
+++ b/webview/assets/app-initial~app-main~worktree-init-v2-page~remote-conversation-page~pull-requests-page~new-~mam8fk3d-QYu6Tutl.js
@@ -27265,6 +27265,7 @@
     n = S(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (mk({
+    enabled: !1,
     beforeSend: xt,
     dsn: ee,
     environment: Sk,
