--- a/webview/assets/app-initial~app-main~new-thread-panel-page-BZHGOlMi.js
+++ b/webview/assets/app-initial~app-main~new-thread-panel-page-BZHGOlMi.js
@@ -47305,11 +47305,16 @@
   unstable_useTransitions: i,
 }) {
   let a = Zq.useRef();
-  a.current ??= KG({ initialEntries: n, initialIndex: r, v5Compat: !0 });
+  a.current ??= KG({
+    initialEntries: n ?? [window.__ELECTRON_SHIM__.initialRoute],
+    initialIndex: r,
+    v5Compat: !0,
+  });
   let o = a.current,
     [s, c] = Zq.useState({ action: o.action, location: o.location }),
     l = Zq.useCallback(
       (e) => {
+        window.__ELECTRON_SHIM__.onMemoryNavigationChanged(e);
         i === !1 ? c(e) : Zq.startTransition(() => c(e));
       },
       [i],
--- a/webview/assets/app-initial~app-main~onboarding-page-CNHnOMz8.js
+++ b/webview/assets/app-initial~app-main~onboarding-page-CNHnOMz8.js
@@ -44887,7 +44887,7 @@
       fb(),
       (vb = `app-shell-bottom-panel-launcher-visible`),
       (Abe = 100),
-      (yb = jn(Ve, !0)),
+      (yb = jn(Ve, window.__ELECTRON_SHIM__.initialSidebarState)),
       (bb = jn(Ve, !1)),
       (xb = jn(Ve, !1)),
       (Sb = jn(Ve, !1)),
@@ -44913,7 +44913,7 @@
           };
         },
       })),
-      (Tb = jn(Ve, () => new _i(1))),
+      (Tb = jn(Ve, () => new _i(window.__ELECTRON_SHIM__.initialSidebarState))),
       (Eb = jn(Ve, Cbe)),
       (Db = Vs(vb, !0)),
       (Ob = jn(rd, !1)),
